R/gg-plots.R

```diff
@@ -1,17 +1,13 @@
 # add global variable
-if (getRversion() >= "2.15.1") {
-  utils::globalVariables(unique(c(
-    "labelp", # cor plot
-    c("x"), # facetdensitystrip plot
-    c("x"), # density diagonal plot
-    c("x", "y", "lab"), # internal axis plot
-    c("x", "y", "result", "freq"), # fluctuation plot
-    c("weight") # ggally_summarise_by
-  )))
-}
-
-
-
+utils::globalVariables(c(
+  c("labelp"), # cor plot
+  c("x"), # facetdensitystrip plot
+  c("x"), # density diagonal plot
+  c("x", "y", "lab"), # internal axis plot
+  c("x", "y", "result", "freq"), # fluctuation plot
+  c("weight"), # ggally_summarise_by
```

Comment:
Is it possible to remove these globalVariable declarations? (and in other files as well)